src/Tribe/Customizer/Text.php

```diff
@@ -0,0 +1,272 @@
+<?php
+/**
+ * The Events Calendar Customizer Section: Text.
+ *
+ * @package The Events Calendar
+ * @since   TBD
+ */
+
+// Don't load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+final class Tribe__Events__Customizer__Text extends Tribe__Customizer__Section {
+
+	/**
+	 * Object instance.
+	 *
+	 * @return mixed|object|Tribe__Container The instance of the requested class.
+	 */
+	public static function instance() {
+		return tribe( 'tec.customizer.text' );
+	}
+
+	/**
+	 * Grab the CSS rules template.
+	 * 
+	 * @since TBD
+	 *
+	 * @param string  $template
+	 * @return string $template
+	 */
+	public function get_css_template( $template ) {
+		$customizer = Tribe__Customizer::instance();
+		$settings   = $customizer->get_option( [ $this->ID ] );
+		if ( $customizer->has_option( $this->ID, 'primary_text_color' ) ) {
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: ad4e023a4481307f91f00af731a4e2f4</sub>